cfgov/v1/wagtail_hooks.py

```diff
@@ -365,21 +366,49 @@ def hide_snippets_menu_item(request, menu_items):
                      if item.url != reverse('wagtailsnippets:index')]
 
 
-# Override list of allowed tags in a RichTextField
-@hooks.register('construct_whitelister_element_rules')
-def whitelister_element_rules():
-    allow_html_class = attribute_rule({
-        'class': True,
-        'itemprop': True,
-        'itemscope': True,
-        'itemtype': True,
-    })
-
-    allowed_tags = ['aside', 'h4', 'h3', 'p', 'span',
-                    'table', 'tr', 'th', 'td', 'tbody', 'thead', 'tfoot',
-                    'col', 'colgroup']
-
-    return {tag: allow_html_class for tag in allowed_tags}
+if wagtail.VERSION < (2, 0):
+    # Override list of allowed tags in a RichTextField
+    @hooks.register('construct_whitelister_element_rules')
+    def whitelister_element_rules():
+        allow_html_class = attribute_rule({
+            'class': True,
+            'itemprop': True,
+            'itemscope': True,
+            'itemtype': True,
+        })
+
+        allowed_tags = ['aside', 'h4', 'h3', 'p', 'span',
+                        'table', 'tr', 'th', 'td', 'tbody', 'thead', 'tfoot',
+                        'col', 'colgroup']
+
+        return {tag: allow_html_class for tag in allowed_tags}
+
+
+# construct_whitelister_element_rules are depricated in Wagtail 2.0
+# https://docs.wagtail.io/en/stable/releases/
+# 2.0.html#construct-whitelister-element-rules-hook-is-deprecated
+if wagtail.VERSION >= (2, 0):
+    @hooks.register('register_rich_text_features')
+    def register_span_feature(features):
+        allow_html_class = attribute_rule({
+            'class': True,
+            'itemprop': True,
+            'itemscope': True,
+            'itemtype': True,
+        })
+
+        allowed_tags = ['span']
+
+        # register a feature 'span'
+        # which whitelists the <span> element
+        features.register_converter_rule('editorhtml', 'span', [
+            WhitelistRule('span', allow_without_attributes),
```

Comment:
We need to use `allow_html_class` here to allow those.

```suggestion
            WhitelistRule('span', allow_html_class),
```